crypto/random/seed/GetEntropyRandomSeed.h

```diff
@@ -21,6 +21,7 @@
 #include "util/Linker.h"
 
 #ifndef _WIN32
```

Comment:
Changing this to ifndef win32 + changing make.js to define win32 for all windows builds should remove the need to check twice